### PR TITLE
Configuración para instalar yarn en el readme

### PR DESCRIPTION
**Pull request by gajamo**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: #5 

## Does this PR introduce a breaking change?
```
[] Yes
[x] No
```

## Other information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,35 @@
...
 # Flexbox playground 
-[![All Contributors](https://img.shields.io/badge/all_contributors-9-orange.svg?style=flat-square)](#contributors)
+[![All Contributors](https://img.shields.io/badge/all_contributors-10-orange.svg?style=flat-square)](#contributors)
+
+
+# How to install yarn
+
+## Install yarn on Windows
+
+There are three options for installing Yarn on Windows, you can fine <a href="https://yarnpkg.com/lang/en/docs/install/#windows-stable" target="_blank">here</a>
```
Comment (alejandronanez) [inline README.md]: Woops, typo here: This should be `you can find` instead of `you can fine` :)

Comment (gajamo) [reply README.md]: Ok Gracias, ya corregir el tema del fine, adicional agregue como instalar las dependencias con yarn y probar si se tiene instalado yarn con el yarn --version, gracias

**Comment by gajamo** [comment]
Delete  package-lock.json  and add to .gitignore, and add link to Yarn's docs official site.

**Comment by alejandronanez** [comment]
Merged, thanks @gajamo !

**Pull request merged**